pxr/usd/plugin/usdMtlx/CMakeLists.txt

```diff
@@ -19,10 +19,8 @@ pxr_plugin(usdMtlx
         usdShade
         usdUI
         usdUtils
-        ${MATERIALX_LIBRARIES}
-
-    INCLUDE_DIRS
-        ${MATERIALX_INCLUDE_DIRS}
+        MaterialXCore
```

Comment:
It allows linking only the libraries that are in use instead of all libraries.
Also, with this modern CMake implementation, the required include dirs (and also the required compiler definitions) gets added on a per-library basis.